### ISSUE TITLE
Keeweb connection fails on Ubuntu 21.10

### ISSUE THREAD

**Issue by johndevs** [keeweb/keeweb-connect#33]
When trying to enable the plugin for firefox in Ubuntu 21.10 you get this error:

**Error:**
`2021-11-23T18:34:43.468Z [browser-extension-connector] Error installing extension: Error: ENOENT: no such file or directory, unlink '/home/john/.mozilla/native-messaging-hosts/net.antelle.keeweb.keeweb_connect.json'`

This is because in Ubuntu 21.10+  Firefox will be installed via Snap instead of via apt (see https://www.omgubuntu.co.uk/2021/09/ubuntu-makes-firefox-snap-default ) 

When installing via snap the `.mozilla` folder is no longer located under the `$HOME/.mozilla` directory but under  `$HOME/snap/firefox/common/.mozilla`. This breaks the plugin.

**Workaround:**
```
sudo snap remove firefox
sudo apt install firefox
```

Using the non-snap version of firefox works, using the snap version does not.

**Comment by antelle** [comment]
How do other apps install itself as native hosts with the snap version of Firefox?

**Comment by johndevs** [comment]
I don´t think it is possible under flatpak due to sandboxing according to https://bugzilla.mozilla.org/show_bug.cgi?id=1621763 (KeePassXC seems to have the same issue)

**Comment by johndevs** [comment]
How KeepassXC worked around the issue for their plugin, maybe the same approach could be taken?

https://discourse.flathub.org/t/how-to-run-firefox-and-keepassxc-in-a-flatpak-and-get-the-keepassxc-browser-add-on-to-work/437?u=rugk
